src/main/webapp/app/js/goal-page.js

```diff
@@ -35,12 +36,14 @@ function onLoad() {
             return false;
         }
     }).then(function (goal) {
-        console;
         w3.displayObject("title", goal);
         w3.displayObject("goal-fields", goal);
         w3DisplayData("tags-list", goal);
+        document.getElementById("goal-deadline").setAttribute("value", goal.deadlineDate);
         getComments();
     });
+    isGoalEditable();
```

Comment:
The `is` prefix is usually used with functions that return boolean values - however, this function does not return anything. In this context, its name is confusing - a better name would be something like `ensureGoalIsEditable()`.

Reply:
Renamed void js function.